enclave_wrangler/config.py

```diff
@@ -12,6 +12,10 @@
 ENV_FILE = os.path.join(ENV_DIR, '.env')
 TERMHUB_CSETS_DIR = os.path.join(PROJECT_ROOT, 'termhub-csets')
 UPLOADS_DIR = os.path.join(TERMHUB_CSETS_DIR, 'datasets', 'uploads')
+OUTDIR_OBJECTS = os.path.join(TERMHUB_CSETS_DIR, 'objects')
+OUTDIR_DATASETS = os.path.join(TERMHUB_CSETS_DIR, 'datasets')
+OUTDIR_DATASETS_DOWNLOADED = os.path.join(OUTDIR_DATASETS, 'downloads')
+OUTDIR_DATASETS_TRANSFORMED = os.path.join(OUTDIR_DATASETS, 'prepped_files')
```

Comment:
Moved some of these constants from `app.py`, etc, into `enclave_wrangler/config.py`